tests/utils.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+from __future__ import unicode_literals, print_function
+
+import xmltodict
+
+
+def xml_to_dict_unordered(xml):
```

Comment:
This replaces the entire pxxmltest library (it's error-reporting isn't as nice and it only tests entire docs, thanks to [`xmltodict`](https://github.com/martinblech/xmltodict) it's simple; thanks @honkfestival for finding this).